src/wp-includes/class-wp-user-query.php

```diff
@@ -148,6 +152,19 @@ public static function fill_query_vars( $args ) {
 	 *                                             roles. Default empty array.
 	 *     @type string[]     $role__not_in        An array of role names to exclude. Users matching one or more of these
 	 *                                             roles will not be included in results. Default empty array.
+	 *     @type string       $capability          An array or a comma-separated list of capability names that users must match
+	 *                                             to be included in results. Note that this is an inclusive list: users
+	 *                                             must match *each* capability.
+	 *                                             Does NOT work for capabilities not in the database or filtered via `map_meta_cap`.
```

Comment:
Let's use a `@see` reference here, to have it auto-linked on DevHub:
```
Does NOT work for capabilities not in the database or filtered via {@see 'map_meta_cap'}.
```
Same for the other two parameters.